### PR TITLE
add a1 description

### PR DESCRIPTION
**Pull request by jrenaf**

### CODE REVIEW

File: spirit_simulator/a1_description/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.3)
```
Comment (ardalantj) [inline spirit_simulator/a1_description/CMakeLists.txt:1]: General comment: perhaps spirit simulator would no longer make sense as the name of the folder. Maybe we can change it to just simulator? 
@jcnorby Not sure if we want to do it here or in a separate pr

File: spirit_simulator/a1_description/launch/a1_rviz.launch [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<launch>
+
+    <arg name="user_debug" default="false"/>
+    
+    <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find a1_description)/xacro/robot.xacro'
+            DEBUG:=$(arg user_debug)"/>
+    
+    <!-- for higher robot_state_publisher average rate-->
```
Comment (ardalantj) [inline spirit_simulator/a1_description/launch/a1_rviz.launch:8]: remove?

File: spirit_simulator/a1_description/package.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package format="2">
+    <name>a1_description</name>
+    <version>0.0.0</version>
+    <description>The a1_description package</description>
+
+    <maintainer email="jimingre@andrew.cmu.edu">unitree</maintainer>
+    <license>TODO</license>
```
Comment (ardalantj) [inline spirit_simulator/a1_description/package.xml:8]: Is this meant to be left as a TODO?

File: spirit_simulator/a1_description/CMakeLists.txt (hunk 1 above)
Comment (jcnorby) [reply spirit_simulator/a1_description/CMakeLists.txt:1]: It isn't called spirit_simulator anymore, it should be quad_simulator. these files should be relocated accordingly.

File: spirit_simulator/a1_description/launch/check_joint.rviz [hunk 4]
```diff
@@ -0,0 +1,265 @@
+Panels:
```
Comment (jcnorby) [inline spirit_simulator/a1_description/launch/check_joint.rviz:1]: Do we need this file?

File: spirit_simulator/a1_description/package.xml (hunk 3 above)
Comment (jcnorby) [reply spirit_simulator/a1_description/package.xml:8]: Likewise, we should check the license on this package to see if there is any alterations we need to make, although that could likely be a separate PR.

File: spirit_simulator/a1_description/sdf_mesh/a1_new.sdf [hunk 5]
```diff
@@ -0,0 +1,1128 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jcnorby) [inline spirit_simulator/a1_description/sdf_mesh/a1_new.sdf:1]: We should only have one sdf/urdf for our repo, no need to keep the outdated one. Those files should be named <robot_name>.urdf/sdf so that they work with our launchfiles.

**Review by jcnorby — CHANGES_REQUESTED** [review]

File: spirit_simulator/a1_description/sdf_mesh/a1_new.sdf (hunk 5 above)
Comment (jrenaf) [reply spirit_simulator/a1_description/sdf_mesh/a1_new.sdf:1]: Ah, we might ignore this pull request at this point. I submit another one with both a1 description added and other changes included to make a1 work.
